### DISCUSSION TITLE
Close discussions without deleting them

### DISCUSSION THREAD

**Discussion by audrow** [community/community#3136] [Discussions] [answered]
I would like to do something to remove resolved discussions from the board. Something like how issues or PRs can be closed would be great.

My current workaround is to have an archive category. This isn't ideal because it is still visible in all categories and I lose the information about which category the discussion came from (unless I make an archive for each category).

**Comment by abravalheri** [comment]
@audrow is this related to https://github.com/github/feedback/discussions/3097? Maybe these 2 discussions could "join forces"?

**Reply by audrow** [reply]
Yes, they seem like the same thing. How do I close this in favor of that one? :laughing: Do you think setting your reply as an answer makes sense?

**Reply by abravalheri** [reply]
I have absolutely no idea 😝 
Maybe that is another feedback for them, mark discussion as duplicate...

**Comment by jricciardi** [comment, marked as answer]
👋 Hey there @audrow! We've updated https://github.com/orgs/community/discussions/3097 with an announcement that you are now able to close Discussions without deleting them.

In fact, you can even now add a 'close' reason — so, I'm going to go ahead and close this Discussion as a duplicate 😄 

 Please read more about the feature here: https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/

**Reply by audrow** [reply]
Great, thank you!
